accepted/future-releases/generalized-typedef-2018/implementation-plan.md

```diff
@@ -0,0 +1,60 @@
+# Implementation Plan for the Q1 2019 Generalized Type Alias Feature
+
+Relevant documents:
+ - [Tracking issue](https://github.com/dart-lang/language/issues/115)
+ - [Full proposal](https://github.com/dart-lang/language/blob/master/accepted/future-releases/generalized-typedef-2018/feature-specification.md)
```

Comment:
I would change 'Full proposal' to 'Feature specification'

Reply:
Done.